<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComboUserItemWidget</class>
 <widget class="QWidget" name="ComboUserItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComboUserItemWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="backWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QLabel" name="avatarLabel">
        <property name="minimumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>26</width>
          <height>26</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>icon</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="closeToolButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../inst.qrc">
          <normaloff>:/images/Icon_07.png</normaloff>:/images/Icon_07.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../inst.qrc"/>
 </resources>
 <connections/>
</ui>
